interpreter/cling/test/NullDeref/If.C

```diff
@@ -0,0 +1,26 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+// RUN: cat %s | %cling -Xclang -verify
+//This file checks an if statement for null prt dereference.
+
+int* p = 0;
+
+if (*p) {} // expected-warning {{null passed to a callee that requires a non-null argument}}
```

Comment:
*If* `p` is `0x1` then the warning `null passed` nor `nullptr passed` are wrong.